prowler/providers/gcp/exceptions/exceptions.py

```diff
@@ -0,0 +1,90 @@
+from prowler.exceptions.exceptions import ProwlerException
+
+
+class GCPBaseException(ProwlerException):
+    """Base class for GCP Errors."""
+
+    GCP_ERROR_CODES = {
+        (1925, "GCPCloudResourceManagerAPINotUsedError"): {
+            "message": "Cloud Resource Manager API not used",
+            "remediation": "Enable the Cloud Resource Manager API for the project.",
+            "file": "{file}",
+            "provider": "GCP",
```

Comment:
The same as with the other providers, probably we don't need this here.

Reply:
Removed ✅